pkgs/applications/misc/candle/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, qtbase, qtserialport, qmake }:
+
+stdenv.mkDerivation rec {
+  name = "candle-${version}";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner  = "Denvi";
+    repo   = "Candle";
+    rev    = "v${version}";
+    sha256 = "1gpx08gdz8awbsj6lsczwgffp19z3q0r2fvm72a73qd9az29pmm0";
+  };
+
+  buildPhase = ''
+    cd src
+    qmake
+    buildPhase
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp Candle $out/bin/
+  '';
```

Comment:
Make sure to run the ```pre``` and ```post``` hooks, so others can easily override and you can combine the ```mkdir/cp``` calls into one ```install``` like this:

```
    runHook preInstall

    install -Dm755 Candle $out/bin/candle

    runHook postInstall
```

It would be nice if you installed the README as well as well as any other relevant documentation (if any).

Reply:
Will the generated makefile not have install section?

Reply:
> Will the generated makefile not have install section?

It doesn't.